test/puppetlabs/trapperkeeper/authorization/testutils.clj

```diff
@@ -26,3 +31,9 @@
    (new-rule type path :any))
   ([type path method]
    (rules/new-rule type path method 500 "test rule")))
+
+(defn url-encoded-cert
+  [x509-cert]
+  (let [cert-writer (StringWriter.)
+        _ (ssl-utils/cert->pem! x509-cert cert-writer)]
+    (ring-codec/url-encode cert-writer)))
```

Comment:
Oh right, duh :)  This is probably why I see "side-effecting" functions return the input in addition to modifying it, so you can use the function in a pipeline like this